### PR TITLE
Add a new section to create annotation guidelines.

### PR DESCRIPTION
**Pull request by mariarivera30**
Jira RDISCROWD-1873

Describe your changes
Can we provide a simple text input box (actually similar to e.g. the one that is used to write this description here in JIRA) that allows basic formatting of text

### CODE REVIEW

File: templates/projects/task_presenter_editor.html [hunk 1]
```diff
@@ -8,6 +8,11 @@
 <link rel="stylesheet" href="{{url_for('static', filename='vendor/codemirror/codemirror.css')}}"/>
 <link rel="stylesheet" href="{{url_for('static', filename='vendor/codemirror/addons/display/fullscreen.css')}}"/>
 <link rel="stylesheet" href="{{url_for('static', filename='vendor/codemirror/addons/dialog/dialog.css')}}"/>
+<link rel="stylesheet" href="{{url_for('static', filename='vendor/summernote/summernote.css')}}"/>
+<link  href="{{url_for('static', filename='vendor/summernote/font/summernote.eot')}}"/>
+<link  href="{{url_for('static', filename='vendor/summernote/font/summernote.ttf')}}"/>
+<link  href="{{url_for('static', filename='vendor/summernote/font/summernote.wolf')}}"/>
```
Comment (markroseberry) [inline templates/projects/task_presenter_editor.html]: Shouldn't the file extension be `.woff`?

File: templates/projects/task_presenter_editor.html [hunk 2]
```diff
@@ -282,6 +322,11 @@ <h4 class="modal-title"><strong>{{project.name}}</strong>: {{ _('Task Presenter
     }
   }
 });
+
+setInterval(function() {
+  editor.refresh();
```
Comment (markroseberry) [inline templates/projects/task_presenter_editor.html]: What does this do? Why is it needed?

Comment (mariarivera30) [reply templates/projects/task_presenter_editor.html]: The code mirror not working properly but Ill use the event when clicking the tab.

File: templates/projects/task_presenter_editor.html (hunk 1 above)
Comment (mariarivera30) [reply templates/projects/task_presenter_editor.html]: be careful with the wolf!

**Comment by kbecker42** [comment]
Just two small suggestions to consider:

1. Adjusting the height of the "Annotation Guidelines" text editor control to be inline with the left-side vertical menu (i.e., the menu "Info, Tasks, Statistics, Performance Stats, ..."). This could help the layout of the task presenter page to flow nicely. See screenshots below.

2. Storing the active tab "Annotation Guidelines" or "Task Presenter" for the project between page-loads (such as in `localStorage`). When the user accesses the page, the last used tab automatically displays. This could make it easier to continue where you left off when editing Task Presenter code, after having already completed the Annotation Guidelines. Or, perhaps we could just assume if a value is already present in Annotation Guidelines, then we automatically default to the Task Presenter tab.

```css
.note-editor.note-frame.panel {
  height: 427px;
}
```

## Original

align-off

## Proposed

align

**Comment by mariarivera30** [comment]
> Just two small suggestions to consider:
> 
> 1. Adjusting the height of the "Annotation Guidelines" text editor control to be inline with the left-side vertical menu (i.e., the menu "Info, Tasks, Statistics, Performance Stats, ..."). This could help the layout of the task presenter page to flow nicely. See screenshots below.
> 2. Storing the active tab "Annotation Guidelines" or "Task Presenter" for the project between page-loads (such as in `localStorage`). When the user accesses the page, the last used tab automatically displays. This could make it easier to continue where you left off when editing Task Presenter code, after having already completed the Annotation Guidelines. Or, perhaps we could just assume if a value is already present in Annotation Guidelines, then we automatically default to the Task Presenter tab.
> 
> ```css
> .note-editor.note-frame.panel {
>   height: 427px;
> }
> ```
> 
> ## Original
> align-off
> 
> ## Proposed
> align

For the first point in my next PR I'm adding the update button and the size of the markup might be same as code mirror. To make them look similar.

For the second point,  for the tabs that is handle in the next PR too. Im doing it in a different way, but Im keeping the right tab selected.

**Pull request merged**